examples/data_masking/src/getting_started_encrypt_data.py

```diff
@@ -0,0 +1,106 @@
+import os
+
+from aws_lambda_powertools.utilities.data_masking import DataMasking
+from aws_lambda_powertools.utilities.data_masking.provider.kms.aws_encryption_sdk import AwsEncryptionSdkProvider
+
+KMS_KEY_ARN = os.environ["KMS_KEY_ARN"]
+
+def lambda_handler(event, context):
+
+    data = {
+        "id": 1,
+        "name": "John Doe",
+        "age": 30,
+        "email": "johndoe@example.com",
+        "address": {
+            "street": "123 Main St", 
+            "city": "Anytown", 
+            "state": "CA", 
+            "zip": "12345",
+        },
+    }
+
+    encryption_provider = AwsEncryptionSdkProvider(keys=[KMS_KEY_ARN])
+    data_masker = DataMasking(provider=encryption_provider)
+
+    encrypted = data_masker.encrypt(data=data, fields=["email", "address.street"])
+    # encrypted = {
```

Comment:
What is the recommended way to show output for this? Linting is complaining about "Commented out code" for the current way

Reply:
I think we can 1 or examples and its enough. Show this as an extra tab and its ok.